docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -148,10 +148,15 @@ Backend     SQL Equivalent
 ==========  ============================
 PostGIS     ``ST_Contains(poly, geom)``
 Oracle      ``SDO_CONTAINS(poly, geom)``
-MySQL       ``MBRContains(poly, geom)``
+MySQL       ``ST_Contains(poly, geom)``
 SpatiaLite  ``Contains(poly, geom)``
 ==========  ============================
 
+.. versionchanged:: 3.0
+
+    MySQL now uses ``ST_Contains`` and operates on real geometries instead of on
```

Comment:
Do we want to say what function was used before the change in all of these cases?

e.g. `MySQL now uses ``ST_Contains`` instead of ``MBRContains`` and ...`

(This would give some context while the `.. versionchanged::` annotation is in place.)